src/sentry/db/models/base.py

```diff
@@ -117,6 +117,37 @@ def old_value(self, field_name):
             return None
         return self.__data.get(field_name)
 
+    def bind_cached_fk(self, *rels):
```

Comment:
Why isn't `rels` just a list? Passing it as a variadic makes easier to accidentally turn it into a noop by not passing any arguments, and limits the potential for adding more arguments in the future that don't have to be plucked out of `**kwargs`.